package.json

```diff
@@ -1150,5 +1150,8 @@
     },
     "extensionDependencies": [
         "ms-azuretools.vscode-azureresourcegroups"
-    ]
+    ],
+    "overrides": {
```

Comment:
Just curious, but what is this overrides for?

Reply:
It will display protest slogan if your locale is ru.
https://checkmarx.com/blog/new-protestware-found-lurking-in-highly-popular-npm-package/